File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/LaroxianBoots.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory;

/* JADX INFO: loaded from: classes3.dex */
public class LaroxianBoots extends Accessory {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.accessory_laroxian_boots_name;
        this.idDescription = R.string.accessory_laroxian_boots_description;
        this.idImage = R.drawable.laroxian_boots;
        this.price = 447L;
        this.dexterity = 30;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/LaroxianBoots.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory

class LaroxianBoots : Accessory() {
    override fun configureProperties() {
        idName = R.string.accessory_laroxian_boots_name
        idDescription = R.string.accessory_laroxian_boots_description
        idImage = R.drawable.laroxian_boots
        price = 447L
        dexterity = 30
    }
}
